2 - RTB/Documentazione esterna/Piano di Progetto/Piano di Progetto.typ

```diff
@@ -782,16 +783,15 @@ Conseguentemente ai problemi rilevati, sono state individuate le relative soluzi
     columns: 8,
     fill: (col, row) => if col == 0 and row > 8 or row == 0 {rgb("#bbbbbb")},
     [*Membro*], [*Responsabile*], [*Amministratore*],[*Analista*],[*Progettista*], [*Programmatore*], [*Verificatore*],[*Totale*],
-    [Banzato Alessio],  [/],  [/],    [4],    [/],    [/],   [/], [4],
-    [Carraro Riccardo], [3],  [/],    [/],    [/],    [(2)],   [/], [5],
-    [Gardin Giovanni],  [/],  [/],    [4],    [/],    [/],   [/], [4],
-    [Nardo\ Silvio],     [/],  [3(-1)],[/],    [/],    [(2)],   [/], [4],
-    [Oseliero Antonio], [/],  [/],    [/],    [3(-2)],[(2)],   [/], [3],
-    [Todesco Mattia],   [/],  [/],    [/],    [/],    [/],   [3], [3],
-    [Zaccone Rosario],  [/],  [/],    [/],    [3(-2)],[(2)],   [/], [3],
-    [Totale\ ore ruolo], [3],  [2],    [8],    [2],    [8],   [3], [24],
-    [Costo\ orario],     [30,00], [20,00], [25,00], [25,00], [15,00], [15,00], [],
-    [Costo\ ruolo],      [90,00], [40,00], [200,00], [50,00], [120,00], [45,00], [545,00]
+    [Banzato Alessio],  [/],     [/],      [4],     [/],     [/],      [/],     [4],
+    [Carraro Riccardo], [3],     [/],      [/],     [/],     [2],      [/],     [5],
+    [Gardin Giovanni],  [/],     [/],      [4],     [/],     [/],      [/],     [4],
+    [Nardo\ Silvio],    [/],     [2(-1)],  [/],     [/],     [2],      [/],     [4(-1)],
+    [Oseliero Antonio], [/],     [/],      [/],     [1(-1)], [2],      [/],     [3(-1)],
+    [Todesco Mattia],   [/],     [/],      [/],     [/],     [/],      [3],     [3],
+    [Totale\ ore ruolo],[3],     [3],      [8],     [2],     [8],      [3],     [26],
```

Comment:
il totale ore dei programmatori credo sia sbagliato (2+2+2=6 e non 8)

Reply:
I conti erano giusti, il problema era che per sbaglio non avevo riportato Rosario in tabella

Reply:
ok top